contrib/confluentinc/confluent-kafka-go/kafka/kafka.go

```diff
@@ -40,18 +41,26 @@ func NewProducer(conf *kafka.ConfigMap, opts ...Option) (*Producer, error) {
 	if err != nil {
 		return nil, err
 	}
-	return WrapProducer(p, opts...), nil
+
+	return WrapProducerWithConfig(p, conf, opts...), nil
+}
+
+// A kafkaConfig struct holds information from the kafka config for span tags
+type kafkaConfig struct {
+	bootstrapServers string
 }
 
 // A Consumer wraps a kafka.Consumer.
 type Consumer struct {
 	*kafka.Consumer
+	kafkaConfig
 	cfg    *config
 	events chan kafka.Event
 	prev   ddtrace.Span
 }
 
 // WrapConsumer wraps a kafka.Consumer so that any consumed events are traced.
+// Deprecated: Replaced with WrapConsumerWithConfig such that config information can be used in span tags.
 func WrapConsumer(c *kafka.Consumer, opts ...Option) *Consumer {
 	wrapped := &Consumer{
 		Consumer: c,
```

Comment:
```suggestion
		Consumer: c,
		kafkaCfg: &kafkaConfig{},
```
(to make sure you don't have issues with `nil` pointer)